src/components/panel/panel.js

```diff
@@ -721,6 +768,9 @@ function MdPanelService($rootElement, $rootScope, $injector, $window) {
   /** @private {!Object<string, !MdPanelRef>} */
   this._trackedPanels = {};
 
+  /** @private {!Object<string, {panels: !Array<!MdPanelRef>, maxOpen: number}>} */
```

Comment:
I think this type is wrong now since there are more properties. (Missing `openPanels`.)